src/pages/variantsSearchPage/VariantStats.tsx

```diff
@@ -33,22 +34,22 @@ const VariantStatsContainer = () => {
       <StackLayout className={style.variantStatsContainer}>
         <CountWithIcon
           total={formatCounts(result?.stats?.studiesCount)}
-          Icon={<StudyIcon fill={colors.iconColor} height={30} width={30} />}
+          Icon={<StudyIcon className={style.variantPageIconColor} height={30} width={30} />}
```

Comment:
not necessarily your code but, how about : 

const iconSize = { height={30} width={30} }
...
<StudyIcon className={style.variantPageIconColor} {...iconSize} />